android/engine/src/main/res/drawable/ic_pregnant.xml

```diff
@@ -4,17 +4,7 @@
774H9.773H9.773H9.772H9.772H9.772H9.771H9.771H9.77H9.77H9.769H9.769H9.768H9.768H9.767H9.767H9.766H9.766H9.765H9.765H9.764H9.763H9.763H9.762H9.762H9.761H9.761H9.76H9.759H9.759H9.758H9.758H9.757H9.756H9.756H9.755H9.755H9.754H9.753H9.753H9.752H9.751H9.751H9.75H9.749H9.749H9.748H9.747H9.746H9.746H9.745H9.744H9.744H9.743H9.742H9.741H9.741H9.74H9.739H9.738H9.737H9.737H9.736H9.735H9.734H9.733H9.733H9.732H9.731H9.73H9.729H9.728H9.728H9.727H9.726H9.725H9.724H9.723H9.722H9.721H9.721H9.72H9.719H9.718H9.717H9.716H9.715H9.714H9.713H9.712H9.711H9.71H9.709H9.708H9.707H9.706H9.705H9.704H9.703H9.702H9.701H9.7H9.699H9.698H9.697H9.696H9.695H9.694H9.693H9.692H9.69H9.689H9.688H9.687H9.686H9.685H9.684H9.683H9.681H9.68H9.679H9.678H9.677H9.676H9.674H9.673H9.672H9.671H9.67H9.668H9.667H9.666H9.665H9.663H9.662H9.661H9.66H9.658H9.657H9.656H9.654H9.653H9.652H9.651H9.649H9.648H9.647H9.645H9.644H9.642H9.641H9.64H9.638H9.637H9.636H9.634H9.633H9.631H9.63H9.628H9.627H9.626H9.624H9.623H9.621H9.62H9.618H9.617H9.615H9.614H9.612H9.611H9.609H9.608H9.606H9.605H9.603H9.602H9.6H9.598H9.597H9.595H9.594H9.592H9.59H9.589H9.587H9.586H9.584H9.582H9.581H9.579H9.577H9.576H9.574H9.572H9.571H9.569H9.567H9.565H9.564H9.562H9.56H9.558H9.557H9.555H9.553H9.551H9.55H9.548H9.546H9.544H9.542H9.541H9.539H9.537H9.535H9.533H9.531H9.529H9.528H9.526H9.524H9.522H9.52H9.518H9.516H9.514H9.512H9.51H9.508H9.506H9.504H9.502H9.501H9.499H9.497H9.495H9.493H9.49H9.488H9.486H9.484H9.482H9.48H9.478H9.476H9.474H9.472H9.47H9.468H9.466H9.463H9.461H9.459H9.457H9.455H9.453H9.451H9.448H9.446H9.444H9.442H9.44H9.437H9.435H9.433H9.431H9.428H9.426H9.424H9.422H9.419H9.417H9.415H9.412H9.41H9.408H9.405H9.403H9.401H9.398H9.396H9.394H9.391H9.389H9.387H9.384H9.382H9.379H9.377H9.374H9.372H9.369H9.367H9.365H9.362H9.36H9.357H9.355H9.352H9.35H9.347H9.344H9.342H9.339H9.337H9.334H9.332H9.329H9.326H9.324H9.321H9.319H9.316H9.313H9.311H9.308H9.305H9.303H9.3H9.297H9.295H9.292H9.289H9.287H9.284H9.281H9.278H9.276H9.273H9.27H9.267H9.264H9.262H9.259H9.256H9.253H9.25H9.247H9.245H9.242H9.239H9.236H9.233H9.23H9.227H9.224H9.221H9.219H9.216H9.213H9.21H9.207H9.204H9.201H9.198H9.195H9.192H9.189H9.186H9.183H9.18H9.177H9.173H9.17H9.167H9.164H9.161H9.158H9.155H9.152H9.149H9.145H9.142H9.139H9.136H9.133H9.13H9.126H9.123H9.12H9.117H9.113H9.11H9.107H9.104H9.1H9.097H9.094H9.091H9.087H9.084H9.081H9.077H9.074H9.071H9.067H9.064H9.06H9.057H9.054H9.05H9.047H9.043H9.04H9.036H9.033H9.03H9.026H9.023H9.019H9.016H9.012H9.009H9.005H9.001H8.998H8.994H8.991H8.987H8.984H8.98H8.976H8.973H8.969H8.966H8.962H8.958C8.647,5.833 8.394,5.967 8.246,6.06C8.08,6.164 7.927,6.295 7.793,6.423C7.524,6.68 7.254,7.004 7.023,7.303C6.788,7.607 6.576,7.908 6.424,8.131C6.348,8.243 6.285,8.337 6.242,8.403L6.191,8.481L6.177,8.502L6.174,8.508L6.172,8.51L6.172,8.511L6.172,8.511C6.172,8.511 6.172,8.511 6.875,8.958L6.172,8.511C5.935,8.882 6.029,9.373 6.385,9.632L8.676,11.299ZM8.533,8.082C8.614,8.508 8.685,8.908 8.747,9.289L8.018,8.759C8.117,8.62 8.227,8.471 8.342,8.322C8.406,8.239 8.47,8.159 8.533,8.082Z"
```

Comment:
@ageryck this seems to update an existing icon. How different is it? We might want to notify the non-tech team of the changes

Reply:
I had removed it and added it back, same image same name, zero net effect